nerfstudio/pipelines/base_pipeline.py

```diff
@@ -383,11 +364,11 @@ def load_pipeline(self, loaded_state: Dict[str, Any], step: int) -> None:
             loaded_state: pre-trained model state dict
             step: training step of the loaded checkpoint
         """
-        state = {
-            (key[len("module.") :] if key.startswith("module.") else key): value for key, value in loaded_state.items()
-        }
-        self.model.update_to_step(step)
-        self.load_state_dict(state, strict=True)
+        self.model.load_model_from_pipeline_state(loaded_state, step)
+
+        state = Model.parse_pipeline_state(loaded_state)
+        pipeline_state = {key: value for key, value in state.items() if not key.startswith("_model.")}
+        self.load_state_dict(pipeline_state, strict=False)
```

Comment:
Why is strict=False instead of strict=True now? Any reason in particular? I guess it's since we remove the model keys/values from the state. This could maybe lead to some undesired behavior if, for example, the state for optimized cameras silently fails. Let me know what you think.

Reply:
Thanks for the review. So I know it looks like I did, and when proofreading I had the same question, but I actually didn't :).

There used to be an override of the method `load_state_dict` in `Pipeline` that would parse the model and then the pipeline itself by calling `super().load_state_dict(pipeline_state, strict=False)` at line 127. Since the model loading logic was moved to the `Model` class, we are left here with only `super().load_state_dict(pipeline_state, strict=False)` in the override and thus I have removed the override and I'm calling the torch `load_state_dict` directly here (the same one that previously was line 127). Hence, since the one line 127 was called with False, I've left the same logic here, but the git diff makes it look like I changed the call to the override :S